pom.xml

```diff
@@ -9,7 +9,7 @@
     </parent>
     <groupId>io.jenkins.plugins</groupId>
     <artifactId>macstadium-orka</artifactId>
-    <version>2.05-SNAPSHOT</version>
+    <version>2.0-SNAPSHOT</version>
```

Comment:
@frosemond the original value was `2.05-SNAPSHOT`. We should use it.
Not `2.0-SNAPSHOT`